README.md

```diff
@@ -16,6 +16,7 @@ an event loop, it can be used with this library.
 **Table of Contents**
 
 * [Usage](#usage)
+    * [async()](#async)
```

Comment:
This currently links to `Async` at the top of the page

Reply:
Very good catch! :+1:

I've updated the PR to rename the top heading from "Async" to "Async Utilities" so that we can consistently refer to the function documentation using their function names as a fragment.

Linking to "#async-1" would have been an alternative, but imho makes this both more inconsistent and confusing for consumers when linking to a function. The README automatically gets a "#readme" fragment, so there's little point in linking to "#async" as the top heading, especially given we use use the same README also for generating individual pages on the website where the top heading does not get linked at all.

I've checked this in all other components and the website. It's not perfect, but seems like the best tradeoff at this point. Thanks for spotting!

Reply:
Very good catch! +1

I've updated the PR to rename the top heading from "Async" to "Async Utilities" so that we can consistently refer to the function documentation using their function names as a fragment.

Linking to "#async-1" would have been an alternative, but imho makes this both more inconsistent and confusing for consumers when linking to a function. The README automatically gets a "#readme" fragment, so there's little point in linking to "#async" as the top heading, especially given we use use the same README also for generating individual pages on the website where the top heading does not get linked at all.

I've checked this in all other components and the website. It's not perfect, but seems like the best tradeoff at this point. Thanks for spotting!